documents/tree.go

```diff
@@ -105,6 +105,14 @@ func (cd *CoreDocument) DocumentSaltsFunc() func(compact []byte) ([]byte, error)
 	}
 }
 
+// FingerprintSaltsFunc returns an empty function
+func (cd *CoreDocument) FingerprintSaltsFunc() func(compact []byte) ([]byte, error) {
```

Comment:
I would call this `ZeroSaltsFunc()`

Reply:
this should not be method on Coredocument but rather a function.